source/addComment.js

```diff
@@ -1,3 +1,22 @@
 $(document).ready(function(){
-  // add your code here
+
+  $('#new_comment_button').click(function() {
+    $('#new_comment_button').hide()
+    $('body').append('<form id="new_comment">Comment:<br><textarea rows="4" cols="50" name="body" value="body"></textarea><br><br>Name:<br><input id="authorName" type="text" name="name"><input type="submit" value="Submit"></form>')
+    $('#new_comment').show()
+
+    $('#new_comment').submit(function(event) {
```

Comment:
I would move this function outside of the click function. Allowing it to be it's own event handler